src/slacko.ml

```diff
@@ -317,31 +317,32 @@ type file_obj = {
 
   size: int;
 
-  url: string;
-  url_download: string;
+  (* These two are deprecated and appear to be gone. *)
+  (* url: string; *)
+  (* url_download: string; *)
```

Comment:
If they are gone, then we can safely remove them, no need to retain them as comments, a note in the commit message should suffice.